Podcast/ListeningHistoryViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  ListeningHistoryViewController.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/24/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class ListeningHistoryViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, ListeningHistoryTableViewCellDelegate {
+    
+    ///
+    /// Mark: Constants
+    ///
+    var lineHeight: CGFloat = 3
+    var topButtonHeight: CGFloat = 30
+    var topViewHeight: CGFloat = 60
+    
+    ///
+    /// Mark: Variables
+    ///
+    var listeningHistoryTableView: UITableView!
+    var episodes: [Episode] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .podcastWhiteDark
+        title = "Listening History"
+        
+        //tableview
+        guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+        listeningHistoryTableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height - appDelegate.tabBarController.tabBarHeight))
```

Comment:
Don't shrink the table view size, make this the full view height and then add the tab bar height into the tableView's content Insets on the bottom. That way we get the nice translucent effect on the tab bar